packages/server/api/routes/orderItems.router.js

```diff
@@ -0,0 +1,63 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+
+// controllers
+const orderItemsController = require('../controllers/orderItems.controller');
+
+router.put('/:orderID/:movieID', (req, res) => {
+  orderItemsController
+    .addMovieToCart(req.params.orderID, req.params.movieID)
+    .then((result) => {
+      res.status(200).json(result);
+    })
+    .catch((error) => {
+      res.status(400).send('Bad request').end();
+      res.status(500).json({ error: error.message });
+    });
+});
+
+router.delete('/:userID/:movieID/:orderID', (req, res) => {
+  orderItemsController
+    .removeMovieFromCart(
+      req.params.userID,
+      req.params.movieID,
+      req.params.orderID,
+    )
+    .then((result) => {
+      if (result === 0) {
+        res.status(404).send("Can't find the item that you request to remove.");
+      } else {
+        res.status(200).json({
+          success: true,
+          message: 'Removed movie from cart sucessfully',
+        });
+      }
+    })
+    .catch((error) => {
+      res.status(400).send('Bad request').end();
+      res.status(500).json({ error: error.message });
```

Comment:
Dont see point in having these statements?

Reply:
**res.status(400).json(error.sqlMessage)** I am thinking to replace this
I found when I request this endpoint with wrong id, its keep running in postman if I didnt return **res** and I think its quit useful  to use sqlMessage for for developer to debug the code 
What do you think?

Reply:
Thanks okay when troubleshooting you can console log these messages and  in some cases you might even throw sql exception as error message. But if you intend to cleanup later then it's best to do before doing merge

Reply:
but why you didnt see the point that I return those statement in catch block?

I dont know how to clean it up yet so I want to go with something simple for now :grin:

Reply:
It's more like two different status's being set and returned which I see as problem. It will also ways be first one with status 400 that would be sent back. So you might delete second some with status code 500

Reply:
Ah... I thought you mean return nothing 😁 
I have fixed my code now, Hope its fine now but let me know if I need to changes something else